tests/test_exacttarget_start_date.py

```diff
@@ -0,0 +1,112 @@
+
+        self.assertGreaterEqual(sum(sync_record_count_1.values()), sum(sync_record_count_2.values()))
+
+        for stream in expected_streams:
+            with self.subTest(stream=stream):
+
+                # expected values
+                expected_primary_keys = self.expected_primary_keys()[stream]
+                expected_replication_keys = self.expected_replication_keys()[stream]
+
+                # collect information for assertions from syncs 1 & 2 base on expected values
+                record_count_sync_1 = sync_record_count_1.get(stream, 0)
+                record_count_sync_2 = sync_record_count_2.get(stream, 0)
+                primary_keys_list_1 = [tuple(message.get('data').get(expected_pk) for expected_pk in expected_primary_keys)
+                                       for message in synced_records_1.get(stream, {}).get('messages')
+                                       if message.get('action') == 'upsert']
+                primary_keys_list_2 = [tuple(message.get('data').get(expected_pk) for expected_pk in expected_primary_keys)
+                                       for message in synced_records_2.get(stream, {}).get('messages')
+                                       if message.get('action') == 'upsert']
+
+                primary_keys_sync_1 = set(primary_keys_list_1)
+                primary_keys_sync_2 = set(primary_keys_list_2)
+
+                if self.is_incremental(stream):
+                    # Expected bookmark key is one element in set so directly access it
+                    start_date_keys_list_1 = [message.get('data').get(next(iter(expected_replication_keys))) for message in synced_records_1.get(stream).get('messages')
+                                            if message.get('action') == 'upsert']
+                    start_date_keys_list_2 = [message.get('data').get(next(iter(expected_replication_keys))) for message in synced_records_2.get(stream).get('messages')
+                                            if message.get('action') == 'upsert']
+
+                    start_date_key_sync_1 = set(start_date_keys_list_1)
+                    start_date_key_sync_2 = set(start_date_keys_list_2)
+
+                    # Verify bookmark key values are greater than or equal to start date of sync 1
+                    for start_date_key_value in start_date_key_sync_1:
+                        self.assertGreaterEqual(self.dt_to_ts(start_date_key_value), start_date_1_epoch)
+
+                    # Verify bookmark key values are greater than or equal to start date of sync 2
+                    for start_date_key_value in start_date_key_sync_2:
+                        self.assertGreaterEqual(self.dt_to_ts(start_date_key_value), start_date_2_epoch)
+
+                    # Verify the number of records replicated in sync 1 is greater than the number
+                    # of records replicated in sync 2 for stream
+                    self.assertGreaterEqual(record_count_sync_1, record_count_sync_2)
```

Comment:
This assertion needs to be Greater, not GreaterEqual.

Reply:
Updated the assertion.